frontend/fingertips-frontend/components/molecules/Export/export.types.ts

```diff
@@ -8,3 +8,19 @@ export interface ExportDownload {
   canvas?: HTMLCanvasElement;
   svg?: string;
 }
+
+export enum CsvHeader {
+  IndicatorId = 'Indicator Id',
```

Comment:
I've styled this as:
```suggestion
  IndicatorId = 'Indicator ID',
```

I think this matches the spreadsheet too.